src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -113,7 +115,12 @@ private static SnapshotBuilderProgramDataListOption generateSnapshotBuilderProgr
           .programDataOptions(
               List.of(
                   generateSnapshotBuilderProgramDataRangeOption(
-                      1, "person", "year_of_birth", "Year of birth", 0, 100),
+                      1,
+                      PersonConstants.PERSON,
+                      PersonConstants.YEAR_OF_BIRTH,
+                      "Year of birth",
```

Comment:
Maybe this could be an alias constant too?

Reply:
And race, ethnicity, gender_identity maybe

Reply:
I'm going to change these tests up because the real term isn't `ethnicity` but `ethnicity_concept_id`